app/helpers/application_helper.rb

```diff
@@ -51,7 +51,7 @@ def track_analytics?
 
   # @return [String]
   def html_title(module_item)
-    site_title = 'Early years child development training'
+    site_title = service_name
```

Comment:
Great start, you can actually do away with this line a dry this method up further